app/src/main/res/layout/search_widget_extra_small_v1.xml

```diff
@@ -4,15 +4,19 @@
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 
 <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
-        android:id="@id/button_search_widget_new_tab"
-        android:layout_width="match_parent"
-        android:layout_height="50dp"
-        android:background="@drawable/rounded_white_corners"
-        android:layout_gravity="center">
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@id/button_search_widget_new_tab"
+    android:layout_width="match_parent"
+    android:layout_height="50dp"
+    android:background="@drawable/rounded_white_corners"
+    android:layout_gravity="center">
 
     <ImageView
-            android:id="@+id/button_search_widget_new_tab_icon"
-            android:layout_width="40dp"
-            android:layout_height="40dp"
-            android:layout_gravity="center"/>
-</FrameLayout>
\ No newline at end of file
+        android:id="@+id/button_search_widget_new_tab_icon"
+        android:layout_width="50dp"
+        android:layout_height="50dp"
+        android:scaleType="centerInside"
```

Comment:
Good question. I assume because the other duplicated icon was sized to work with the layout. Tbh. there are multiple things in those layouts that were really strange. I think the widget may need some love.